src/main/java/Duke.java

```diff
@@ -1,10 +1,585 @@
+import com.sun.source.util.TaskListener;
+
+import java.io.*;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAdjusters;
+import java.util.ArrayList;
```

Comment:
Just a small note: Good to separate the different packages (e.g. time, util, io) with an empty line :)